src/view/pages/Farmer/tabs/Info/index.tsx

```diff
@@ -66,22 +51,12 @@ class FarmerInformation extends React.Component<OwnProps, OwnState> {
     this.setState(() => ({ mode: newMode }));
```

Comment:
Please use the function argument for setState ie: `this.setState(state => ({ mode: newMode }))`

I realize it makes no difference in this situation, but this is a pretty tricky bug to track down when you actually are accessing state, and the people coming after us will be reflecting our style and will not likely know the gotcha's of the framework